static/svgs/clients/jebbit.svg

```diff
@@ -1 +1 @@
5 2-.5666695 3.966657-1.249996 5.9-2.05.400002-.133334.699999-.2.9-.2.466669 0 .899998.166665 1.3.5.4.333335.6.766664.6 1.3 0 .866671-.399996 1.466665-1.2 1.8-2.133344.933338-4.3499885 1.6833305-6.65 2.25s-4.649988.85-7.05.85c-5.066692 0-9.133318-1.533318-12.2-4.6s-4.6-7.133308-4.6-12.2v-17.8c0-5.066692 1.533318-9.133318 4.6-12.2s7.133308-4.6 12.2-4.6 9.133318 1.533318 12.2 4.6 4.6 7.133308 4.6 12.2v6c0 .933338-.299997 1.7-.9 2.3-.600003.600003-1.366662.9-2.3.9zm25.8-9.5c0-4.00002-1.1499885-7.1499885-3.45-9.45s-5.44998-3.45-9.45-3.45-7.1499885 1.1499885-9.45 3.45-3.45 5.44998-3.45 9.45v5.9h25.8zm48.7 18.1c0 5.066692-1.533318 9.133318-4.6 12.2s-7.133308 4.6-12.2 4.6c-2.533346 0-5.0166545-.1833315-7.45-.55s-4.816655-.983329-7.15-1.85c-1.466674-.533336-2.2-1.566659-2.2-3.1v-64.6c0-.533336.1833315-1 .55-1.4.3666685-.400002.816664-.6 1.35-.6s.999998.199998 1.4.6c.400002.4.6.866664.6 1.4v22.9c1.866676-1.33334 3.899989-2.366663 6.1-3.1s4.466655-1.1 6.8-1.1c5.066692 0 9.133318 1.533318 12.2 4.6s4.6 7.133308 4.6 12.2zm-3.9-18.1c0-4.00002-1.149989-7.1499885-3.45-9.45-2.3000115-2.3000115-5.44998-3.45-9.45-3.45-2.400012 0-4.7166555.433329-6.95 1.3s-4.216658 2.166658-5.95 3.9v37c4.200021 1.33334 8.499978 2 12.9 2 4.00002 0 7.1499885-1.1499885 9.45-3.45 2.300011-2.3000115 3.45-5.44998 3.45-9.45zm49.2 18.1c0 5.066692-1.533318 9.133318-4.6 12.2s-7.133308 4.6-12.2 4.6c-2.533346 0-5.016655-.1833315-7.45-.55-2.433346-.3666685-4.816655-.983329-7.15-1.85-1.466674-.533336-2.2-1.566659-2.2-3.1v-64.6c0-.533336.183331-1 .55-1.4.366668-.400002.816664-.6 1.35-.6s.999998.199998 1.4.6c.400002.4.6.866664.6 1.4v22.9c1.866676-1.33334 3.899989-2.366663 6.1-3.1s4.466655-1.1 6.8-1.1c5.066692 0 9.133318 1.533318 12.2 4.6s4.6 7.133308 4.6 12.2zm-3.9-18.1c0-4.00002-1.149989-7.1499885-3.45-9.45s-5.44998-3.45-9.45-3.45c-2.400012 0-4.716655.433329-6.95 1.3s-4.216658 2.166658-5.95 3.9v37c4.200021 1.33334 8.499978 2 12.9 2 4.00002 0 7.149989-1.1499885 9.45-3.45s3.45-5.44998 3.45-9.45zm16.7-14.6c0-.533336.199998-.9833315.6-1.35.4-.3666685.866664-.55 1.4-.55s.983331.1833315 1.35.55.55.816664.55 1.35v47.5c0 .533336-.183331.999998-.55 1.4s-.816664.6-1.35.6-1-.199998-1.4-.6c-.400002-.400002-.6-.866664-.6-1.4zm2-16.4c-.866671 0-1.6-.299997-2.2-.9-.600003-.600003-.9-1.333329-.9-2.2s.299997-1.6.9-2.2c.6-.600003 1.333329-.9 2.2-.9s1.58333.299997 2.15.9c.56667.6.85 1.333329.85 2.2s-.28333 1.599997-.85 2.2-1.283329.9-2.15.9zm16.4-4.2c0-.533336.183331-1 .55-1.4.366668-.400002.816664-.6 1.35-.6s.999998.199998 1.4.6c.400002.4.6.866664.6 1.4v19.7h12.9c1.266673 0 1.9.599994 1.9 1.8s-.633327 1.8-1.9 1.8h-12.9v34.1c0 2.800014.733326 5.0166585 2.2 6.65s3.299989 2.45 5.5 2.45c.866671 0 1.833328-.0833325 2.9-.25s1.866664-.25 2.4-.25c.466669 0 .883331.166665 1.25.5.366668.333335.55.766664.55 1.3 0 1.000005-.533328 1.599999-1.6 1.8-.933338.133334-1.849996.2499995-2.75.35-.900005.1000005-1.816662.15-2.75.15-3.400017 0-6.183323-1.1166555-8.35-3.35-2.166678-2.2333445-3.25-5.34998-3.25-9.35z" fill="#647982"/></svg>
```

Comment:
@hannakim91 what changed on these svgs? Adding titles? I don't think we want titles on the svgs, since they will show up when you hover them.